specs/BotBuilderOpenSourcePlans.md

```diff
@@ -0,0 +1,85 @@
+# BotBuilder Open Source Engagement
```

Comment:
Bot Builder under the broad sense also includes the Emulator and CLI tools. We can start with the 5 SDK repos as a tactical approach but every single bot builder repo should adhere to the same engagement policies.